dashboard/charts.py

```
@@ -589,31 +641,58 @@ class ChartStyle:
-            "text": "#e5e7eb" if dark_mode else "#64748b",
-            "grid": "rgba(255,255,255,0.1)" if dark_mode else "rgba(0,0,0,0.1)",
-            "primary": "#3b82f6" if dark_mode else "#2563eb",
-            "secondary": "#9ca3af" if dark_mode else "#64748b",
-            "alert": "#ef4444" if dark_mode else "#dc2626",
-            "llmalert": "#f97316" if dark_mode else "#fb923c",
-            "change": "#fb923c" if dark_mode else "#f97316",
+            "background": "rgba(15,23,42,0.0)" if dark_mode else "rgba(255,255,255,0.0)",
+            "background_solid": "#111827" if dark_mode else "#ffffff",
+            "text": "#dbe4f0" if dark_mode else "#334155",
+            "muted_text": "#94a3b8" if dark_mode else "#64748b",
+            "grid": "rgba(148,163,184,0.12)" if dark_mode else "rgba(148,163,184,0.12)",
```

Ruff (0.15.10)

[warning] Useless `if`-`else` condition

(RUF034)